docs/source/guides/staking_guide.rst

```diff
@@ -114,13 +114,22 @@ off).
 
 If you don't have a hardware wallet, you can create a software one:
 
+Whilst running the initialized node:
```

Comment:
Maybe put this line together with the previous one:

"If you don't have a hardware wallet, you can create a software one. Whilst running the initialized node:"